### DISCUSSION TITLE
My page on Github Pages is only showing my text and some effects when deployed

### DISCUSSION THREAD

**Discussion by morganwestcoding** [community/community#52405] [Other Feature Feedback, Questions, & Ideas] [answered]
Question

I previously was having a blank screen but I fixed the path and deployed and that seemed to get text and some effects to populate. Im trying to get the images and my fonts to populate now. I originally had my css being imported through my Js file so maybe it could be  path issue can someone help me resolve. Heres the repo:
https://github.com/morganwestcoding/morgan-west-portfolio

page is being deployed  from the gh pages branch!

**Comment by SilasRodrigues19** [comment]
Hey @morganwestcoding 

Use `"./"` instead of just using `"/"` in your assets path:

![image](https://user-images.githubusercontent.com/47532703/230805669-a07ab746-f2c0-496c-a514-66c9bc26c990.png)

**Reply by morganwestcoding** [reply]
Silas thank you so much for this input do you know what could preventing the css?

**Reply by SilasRodrigues19** [reply]
Same thing, fix your links where they are with `"/"` to `"./"` your CSS will look like this:

```html
<link rel="stylesheet" href="./App.82631418.css">
```

**Reply by morganwestcoding** [reply]
I apologize for the dumb questions but, I see you have this pulled up in inspect, but as far as my current working files in my repo where are you finding those image names on the gh pages branch?

Also that is the css referenced in my index file but why isnt it picking up the index import from the app.js?

**Reply by morganwestcoding** [reply, marked as answer]
> Hey @morganwestcoding
> 
> Use `"./"` instead of just using `"/"` in your assets path:
> 
> ![image](https://user-images.githubusercontent.com/47532703/230805669-a07ab746-f2c0-496c-a514-66c9bc26c990.png)

After updating my index.html paths with this, the problem still persist unfortunately , any other suggestions?

**Reply by morganwestcoding** [reply]
SAL YOURE A GOD IT WORKED :)!
